juriscraper/opinions/united_states/state/colo.py

```diff
@@ -192,34 +205,93 @@ def _extract_cases_from_sub_page(self, html_tree, date_obj):
 
     @classmethod
     def _extract_docket_from_text(cls, text):
+        """Extracts docket from opinion link
+        Also checks for multiple dockets in the same link file
+        One docket:
+            - 2021 CO 27 No.19SC986, Dep't of Nat. Res. v. 5 Star Feedlot, Inc.
+            - 2020 CO 89 No. 19SC354 Denver Health v. Houchin
+            - 2019 CO 30. No. 16SC783. Gow v. People.
```

Comment:
I'd love to have typing and return values added to everything we touch.

Reply:
I'm sorry. I don't understand this comment

Reply:
I think we should be adding type hints to any method that we touch that doesn't have them already. 

so ..

def _extract_docket_from_text(cls, text): 

to something like this.

def _extract_docket_from_text(cls: Site, text: str) -> str:

Im not sure if that's right or not.

Reply:
> I  think we should be adding type hints to any method that we touch 

Yes. The general idea is that we're doing what's called "gradual typing", which means that instead of dropping everything and adding types everywhere for a week, we're doing it bit by bit whenever we touch a file. The idea is that eventually all the important parts (the ones we work on) will have type hints. 

In CourtListener, there's a file in our github actions that has a list of files and modules that are type-hinted. In Juriscraper, I don't think we have mypy set up as part of the CI pipeline yet, but we'll get it in place eventually.